android/modules/ui/src/java/ti/modules/titanium/ui/widget/webview/TiWebViewBinding.java

```diff
@@ -122,15 +123,15 @@ public void destroy()
 
 	private static StringBuilder readResourceFile(String fileName)
 	{
-		InputStream stream = TiWebViewBinding.class.getClassLoader().getResourceAsStream(
-			"ti/modules/titanium/ui/widget/webview/" + fileName);
-		BufferedReader reader = new BufferedReader(new InputStreamReader(stream));
+		InputStream stream = null;
 		StringBuilder code = new StringBuilder();
 		try {
+			stream = KrollAssetHelper.openAsset("webview/" + fileName);
+			BufferedReader reader = new BufferedReader(new InputStreamReader(stream));
 			for (String line = reader.readLine(); line != null; line = reader.readLine()) {
 				code.append(line + "\n");
 			}
-		} catch (IOException e) {
+		} catch (Exception e) {
```

Comment:
Regarding the WebView JS files, perhaps we should move them from folder...
`assets/webview/`
...to...
`assets/ti.internal/webview/`

This is to avoid collision with the app dev's own assets or the assets belong to an Android library. What do you think?